docs/docs/using-graphql-playground.md

```diff
@@ -18,6 +18,18 @@ To access this experimental feature utilizing GraphQL Playground with Gatsby, ad
 "develop": "GATSBY_GRAPHQL_IDE=playground gatsby develop",
 ```
 
+If you are on **Windows** then you should use:
+
+```
+"develop": "cross-env GATSBY_GRAPHQL_IDE=playground&gatsby develop",
```

Comment:
I don't think we need `&` with cross-env. So probably should be:

```suggestion
"develop": "cross-env GATSBY_GRAPHQL_IDE=playground gatsby develop",
```

Unless I am missing something?